vue-components/src/components/Dialog.vue

```diff
@@ -381,7 +381,7 @@ export default Vue.extend( {
 		position: absolute;
 		inset-block-start: 0;
 		inset-inline-start: 0;
-		transform: translate(-50%, -50%);
+		transform: translate(50%, 50%);
```

Comment:
It looks better now but for some reason still not centered. This might not be the right solution but it solves the overlay issue.